### DISCUSSION TITLE
Debug Mode

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1237] [Q&A] [answered]
What additional information does "Debug Mode" Provide?

**Comment by meetmandeep** [comment, marked as answer]
Vanjaro enables Client Resource Management for each site. "Enable Debug Mode" turns this off temporarily so you can debug client side scripts. In future, we'll use this flag for additional debugging information as needed.

**Reply by scottw12** [reply]
thanks

**Comment by Tychodewaard** [comment]
Hi @meetmandeep 

By accident, I created a website with the debug mode turned on. If I turn it off (which sounds like te desired default) the styling is all over the place.
There is a second logo in the header, the title has much more font-weight, another title is blue instead of white, footer links have underline...what actually happens in debug mode? And can I somehow fix it how it should be?
![image](https://user-images.githubusercontent.com/9091997/150859647-f84b0d1d-20a4-4edb-817b-280d9412b542.png)

![image](https://user-images.githubusercontent.com/9091997/150859363-8e0e9626-a37b-44bc-8591-02990d289272.png)

**Reply by meetmandeep** [reply]
Client Resource Management is enabled with minify JS/CSS and composite files on. It could be a formatting issue in your custom JS or StyleSheet.
